task5.js

```diff
@@ -0,0 +1,43 @@
+class Component {
+  constructor(tag) {
+    this.element = { tag: tag, style: { display: "block" } };
+  }
+  hide() {
+    this.element.style.display = "none";
+  }
+
+  show() {
+    this.element.style.display = "block";
+  }
+}
+class Box extends Component {
+  constructor(element) {
```

Comment:
Название типа `props` или  `options` лучше описывают объект чем `element` 
```suggestion
  constructor(props) {
```